commons-geometry-euclidean/src/test/java/org/apache/commons/geometry/euclidean/oned/Vector1DTest.java

```diff
@@ -120,6 +130,16 @@ public void testNormSq() {
         Assert.assertEquals(9.0, Vector1D.of(-3).normSq(), TEST_TOLERANCE);
     }
 
+    @Test
+    public void testNormSq_unitVectors()
+    {
```

Comment:
Watch the braces in this file. This one should go on the previous line.

Reply:
I thought I caught all those, but I must have missed that file. Should be fixed now :)